### PR TITLE
[Bot] Add prdoc generation

### PR DESCRIPTION
**Pull request by ggwpez**
Add a bot to automatically generate prdocs that have all the crates populated.

### CODE REVIEW

File: .github/workflows/command-prdoc.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        id: gh
+        uses: ./.github/actions/set-up-gh
+        with:
+          pr-number: ${{ inputs.pr }}
+          GH_TOKEN: ${{ github.token }}
+      - name: Generate PrDoc
+        run: |
+          python3 -m pip install -q cargo-workspace PyGithub whatthepatch pyyaml toml
+
+          python3 .github/scripts/generate-prdoc.py --pr "${{ inputs.pr }}" --bump "${{ inputs.bump }}" --audience "${{ inputs.audience }}" --force "${{ inputs.overwrite }}"
```
Comment (mordamax) [inline .github/workflows/command-prdoc.yml:65]: may be you could also alternatively add an option to start it with command?
like here https://github.com/polkadot-fellows/runtimes/pull/406/files#diff-38149785bc1a430631c477eabc9b0970804d2c9433504a67574d0b9d38461ba6R120

```yaml
  issue_comment: # listen for comments on issues
    types: [created]
```

**Review by mordamax — APPROVED** [review]

File: .github/workflows/command-prdoc.yml (hunk 1 above)
Comment (ggwpez) [reply .github/workflows/command-prdoc.yml:65]: I have a `type` set on all inputs to either `number` or `choice` so i dont think that there can be any query injection.

But i would also prefer comments, as outsiders can probably not run CI actions. Not sure why the comment approach was deprecated?

File: .github/scripts/generate-prdoc.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
+"""
```
Comment (bkchr) [inline .github/scripts/generate-prdoc.py:1]:
```suggestion
#!/usr/bin/env python3

"""
```

File: .github/workflows/command-prdoc.yml (hunk 1 above)
Comment (mordamax) [reply .github/workflows/command-prdoc.yml:65]: it wasn't deprecated, we just tried to move to GHA faster/cheaper, and that was one of the fast and secure ways

**Review by EgorPopelyaev — APPROVED** [review]

**Pull request merged**